config/fortify-client.properties

```diff
@@ -0,0 +1,2 @@
+fortify.client.releaseId=73284
```

Comment:
i assume this is grabbed from Fortify, already.

Reply:
Yes, Kaan provided me with this.